dynawo/3rdParty/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+# Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
```

Comment:
Please do not change the copyright of an existing file

Reply:
ok... Question remains should I change a copyright in my (DG) own file ? I won't